contracts/proof_market/mina_account_verifier.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import '@nilfoundation/evm-placeholder-verification/contracts/interfaces/verifier.sol';
+
+
+contract AccountPathVerifier is Ownable {
```

Comment:
Can we please create a new interface  `IEVMEndpointVerifier`   for this under `contracts/interfaces` and have this contract inherit it?